demo/state/common/client.go

```diff
@@ -25,11 +25,11 @@ import (
 )
 
 const (
-	key1 = "key1"
-	key2 = "key2"
-	key3 = "key3"
-	key4 = "key4"
-	key5 = "key5"
+	key1 = "/key1"
```

Comment:
Can we keep it "key1" without the `/`?

Reply:
![1665499281848](https://user-images.githubusercontent.com/75570810/195122374-fa77fae8-0574-4a41-8262-d9f42c823665.jpg)
To support zookeeper, its znode path needs to have '/'. Otherwise, there will be an error that the znode path cannot be found.

Reply:
zk has a configuration option `keyPrefixPath`, is it possible to solve this problem using this option?
If we can't run demo using "key1" without `/`, it's zk component's bug. 
We shouldn't modify the demo since it's weird for new users

Reply:
@zhenjunMa Could u help with this problem?
If we can't run demo using "key1" without "/", I prefer to comment out this state demo test and fix the zk component bug in the future

Reply:
@seeflood sorry.Configuration option keyPrefixPath can solve the problem.